pkg/shared/families/malware/yara/config/config.go

```diff
@@ -15,7 +15,12 @@
 
 package config
 
+import "github.com/openclarity/yara-rule-server/pkg/config"
+
 type Config struct {
-	BinaryPath      string `yaml:"binary_path" mapstructure:"binary_path"`
-	CompiledRuleURL string `yaml:"compiled_rule_url" mapstructure:"compiled_rule_url"`
+	BinaryPath      string              `yaml:"binary_path" mapstructure:"binary_path"`
+	CompiledRuleURL string              `yaml:"compiled_rule_url" mapstructure:"compiled_rule_url"`
+	RuleSources     []config.RuleSource `yaml:"rule_sources" mapstructure:"rule_sources"`
+	YaracPath       string              `yaml:"yarac_path" mapstructure:"yarac_path"`
+	CacheDir        string              `yaml:"cache_dir" mapstructure:"cache_dir"`
```

Comment:
We should probably add a default for this like using: https://pkg.go.dev/os#UserCacheDir otherwise its a hard requirement to specify the directory in the configuration.

Reply:
https://github.com/openclarity/vmclarity/pull/725/commits/8b65ccf9fe1bc97dfc56b8b12ebc40e37892cc09